receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/metadata.yaml

```diff
@@ -12,7 +12,7 @@ attributes:
 
   state:
     description: Breakdown of filesystem usage by type.
-    enum: [free, reserved, used]
+    enum: [free, reserved, used, utilized]
```

Comment:
I don't think `utilized` state is needed. `state` attribute looks redundant for the utilization metric, it always has the same value

Reply:
What should be used then? Could it be just an empty string?

Reply:
done